packages/arb-token-bridge-ui/src/util/networks.ts

```diff
@@ -553,35 +561,48 @@ export function sortChainIds(chainIds: number[]) {
 }
 
 export function getDestinationChainIds(
-  chainId: ChainId,
-  disableTransfersToNonArbitrumChains = false
+  chainId: ChainId | number,
+  {
+    includeLifi = false,
```

Comment:
NIT: I would rename this to `includeLifiEnabledChainPairs` to be consistent with the verbose `disableTransfersToNonArbitrumChains`